tests/chrdev/tests/tests.rs

```diff
@@ -2,6 +2,7 @@ use std::fs;
 use std::io::{Read, Seek, SeekFrom};
 use std::path::PathBuf;
 use std::process::Command;
+use std::os::unix::prelude::*;
```

Comment:
I'm not wild about `*`, any reason not to just import only `FileExt`?

Reply:
Not really, I just feel like preludes are a reasonable thing to `import *` from? https://doc.rust-lang.org/stable/std/os/unix/index.html suggests it.

Reply:
_Drums fingers_ far be it from me to argue with the docs, but I think it makes it much harder to tell why you're importing something.

Reply:
Sure, the other docs export only the single trait, changed :)